app/database/models.py

```diff
@@ -333,8 +337,28 @@ def __repr__(self):
         )
 
 
-# insert language data
+class Parasha(Base):
+    __tablename__ = "parashot"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String, nullable=False)
+    hebrew_name = Column(String, nullable=False)
+    link = Column(String, nullable=False)
+    date = Column(DateTime, nullable=False)
 
+
+class ShoppingProduct(Base):
+    __tablename__ = "shoppingProducts"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String)
```

Comment:
Shouldn't it be 
`name = Column(String, nullable=false)`?

Reply:
i edit that :)